### PR TITLE
0.13: Add Auto Sync Points Section

### PR DESCRIPTION
**Pull request by hymm**
For https://github.com/bevyengine/bevy-website/pull/891 announcement post.

### CODE REVIEW

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 1]
```diff
@@ -52,11 +52,56 @@ TODO.
...
-
-TODO.
+## Automatically insert `apply_deferred` systems
+
+<div class="release-feature-authors">authors: @hymm</div>
+
+A common scheduling issue is that one system needs to see the effects from the 
+commands of another. To this before 0.13 you would manually insert a 
+`apply_deferred` system between the two. Bevy now detects when on system with commands 
+is ordered with another and inserts the apply_deferred for you.
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:62]:
```suggestion
A common scheduling issue is that one system needs to see the effects of the 
commands of another. Before 0.13, you would have to manually insert an 
`apply_deferred` system between the two. Bevy now detects when a system with commands 
is ordered relative to another and inserts the `apply_deferred` for you.
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 2]
```diff
@@ -52,11 +52,56 @@ TODO.
...
+
+```rust
+// Before 0.13
+app.add_systems(
+    Update,
+    (
+        system_with_commands,
+        apply_deferred,
+        another_system
+    ).chain()
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:72]: Nit: I think a linter would want a comma here.
```suggestion
    (
        system_with_commands,
        apply_deferred,
        another_system,
    ).chain()
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 3]
```diff
@@ -52,11 +52,56 @@ TODO.
...
+```
+
+```rust
+// After 0.13
+app.add_systems(
+    Update,
+    (
+        system_with_commands,
+        another_system
+    ).chain()
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:83]:
```suggestion
    (
        system_with_commands,
        another_system,
    ).chain()
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 4]
```diff
@@ -52,11 +52,56 @@ TODO.
...
+);
+```
+
+It also optimizes the auto inserted `apply_deferred` systems by merging them if 
+possible. In most cases it's recommended to remove all manually inserted 
+`apply_deferred` as allowing Bevy to insert and merge these systems as needed will 
+usually be faster.
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:90]:
```suggestion
It also optimizes the automatically inserted `apply_deferred` systems by merging them if 
possible. In most cases, it is recommended to remove all manually inserted 
`apply_deferred` systems, as allowing Bevy to insert and merge these systems as needed will 
usually be faster.
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 5]
```diff
@@ -52,11 +52,56 @@ TODO.
...
+    Update,
+    (
+        (system_1_with_commands, system_2).chain(),
+        (system_3_with_commands, system_4).chain(),
+    )
+);
+```
+
+If this new behavior does not work for you please consult the migration guide. 
+There are a several new APIs for opting out of this.
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:104]:
```suggestion
If this new behavior does not work for you, please consult the migration guide. 
There are several new APIs for opting out of this.
```

**Review by Jondolf — CHANGES_REQUESTED** [review]

**Review by TrialDragon — COMMENTED** [review]
Minor change to the flow of this sentence to make it easier to read.

**Review by TrialDragon — APPROVED** [review]

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 6]
```diff
@@ -57,12 +57,58 @@ TODO.
...
+        (system_3_with_commands, system_4).chain(),
+    )
+);
+```
+
+If this new behavior does not work for you, please consult the migration guide.
+There are several new APIs for opting out of this.
+
+## Input for one-shot systems
 ## Input for One-Shot Systems
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:112]:
```suggestion
## Input for One-Shot Systems
```

**Review by Jondolf — APPROVED** [review]
LGTM!

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 7]
```diff
@@ -57,11 +57,56 @@ TODO.
...
-## Automatically Inserted Sync Points
-
-<div class="release-feature-authors">authors: @TODO</div>
-
-TODO.
+## Automatically Insert `apply_deferred` Systems
+
+<div class="release-feature-authors">authors: @hymm</div>
+
+A common scheduling issue is that one system needs to see the effects of commands
```
Comment (Jondolf) [inline content/news/2024-02-03-bevy-0.13/index.md:64]:
```suggestion
A common scheduling issue is that one system needs to see the effects of commands
```

Comment (hymm) [reply content/news/2024-02-03-bevy-0.13/index.md:64]: Thanks!

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 8]
```diff
@@ -57,11 +57,56 @@ TODO.
...
+queued in another system. Before 0.13, you would have to manually insert an
+`apply_deferred` system between the two. Bevy now detects when a system with commands
+is ordered relative to another and inserts the `apply_deferred` for you.
```
Comment (alice-i-cecile) [inline content/news/2024-02-03-bevy-0.13/index.md:67]:
```suggestion
is ordered relative to other systems and inserts the `apply_deferred` for you.
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 9]
```diff
@@ -57,11 +57,56 @@ TODO.
...
+    Update,
+    (
+        (system_1_with_commands, system_2).chain(),
+        (system_3_with_commands, system_4).chain(),
+    )
+);
+```
+
+If this new behavior does not work for you, please consult the migration guide.
+There are several new APIs for opting out of this.
```
Comment (alice-i-cecile) [inline content/news/2024-02-03-bevy-0.13/index.md:109]:
```suggestion
There are several new APIs that allow you to opt-out.
```

**Pull request merged**